### DISCUSSION TITLE
Share, download, copylink option in PnP Search result as in SharePoint online default search

### DISCUSSION THREAD

**Discussion by vishalgoyal2472** [microsoft-search/pnp-modern-search#4081] [General]
do we have below option available in PnP search result where we can download, share, copy link, and other options for a result.

![image](https://github.com/user-attachments/assets/bb7d0d70-5ce7-4643-908c-30b6a672f5cc)

**Comment by kasperbolarsen** [comment]
Download is an option on some layouts, and copy etc. is provided by the browser.  I haven't seen any option to share and we are actually blocking the web parts from running custom code as it would be a  huge security risk

**Reply by vishalgoyal2472** [reply]
If I want to build this feature can anyone please let me know How I can proceed ?
